### DISCUSSION TITLE
Authelia Docker image not starting

### DISCUSSION THREAD

**Discussion by Steveestrada** [authelia/authelia#8084] [Q&A] [answered]
Hi I set authelia up over 2 years ago and really its been working flawlessly until recently and i just cannot figure out how to get it running again. The docker image will not start here is the logs i am getting any help will be appreciated.

`time="2024-10-21T21:46:50-04:00" level=warning msg="Configuration: configuration keys 'server.host', 'server.port', and 'server.path' are deprecated in 4.38.0 and has been replaced by 'server.address' in the format of '[tcp[(4|6)]://]<hostname>[:<port>][/<path>]' or 'tcp[(4|6)://][hostname]:<port>[/<path>]': you are not required to make any changes as this has been automatically mapped for you to the value 'tcp://0.0.0.0:9091/', but to stop this warning being logged you will need to adjust your configuration, and this configuration key and auto-mapping is likely to be removed in 5.0.0"
time="2024-10-21T21:46:50-04:00" level=warning msg="Configuration: session: option 'domain' is deprecated in v4.38.0 and has been replaced by a multi-domain configuration: this has automatically been mapped for you but you will need to adjust your configuration to remove this message and receive the latest messages"
time="2024-10-21T21:46:50-04:00" level=info msg="Authelia v4.38.16 is starting"
time="2024-10-21T21:46:50-04:00" level=info msg="Log severity set to info"
time="2024-10-21T21:46:50-04:00" level=info msg="Storage schema is being checked for updates"
time="2024-10-21T21:46:50-04:00" level=info msg="Storage schema is already up to date"
time="2024-10-21T21:46:51-04:00" level=error msg="Error occurred running a startup check" error="failed to dial connection: SMTP AUTH failed: 525 5.7.13 We cannot accept this request as your account is deleted. Please contact support@sendinblue.com if you think it's done by mistake." provider=notification

time="2024-10-21T21:46:51-04:00" level=fatal msg="One or more providers had fatal failures performing startup checks, for more detail check the error level logs" providers="[notification]" stack="github.com/authelia/authelia/v4/internal/commands/root.go:150 doStartupChecks\ngithub.com/authelia/authelia/v4/internal/commands/root.go:88  (*CmdCtx).RootRunE\ngithub.com/spf13/cobra@v1.8.1/command.go:985                  (*Command).execute\ngithub.com/spf13/cobra@v1.8.1/command.go:1117                 (*Command).ExecuteC\ngithub.com/spf13/cobra@v1.8.1/command.go:1041                 (*Command).Execute\ngithub.com/authelia/authelia/v4/cmd/authelia/main.go:10       main\ninternal/runtime/atomic/types.go:194                          (*Uint32).Load\nruntime/asm_amd64.s:1700                                      goexit"`

**Comment by Steveestrada** [comment, marked as answer]
This was resolved by updating my smtp info.

**Reply by nightah** [reply]
Yep looks like the culprit was called out here in the logs:

```
SMTP AUTH failed: 525 5.7.13 We cannot accept this request as your account is deleted. Please contact support@sendinblue.com if you think it's done by mistake."
```

Glad it's sorted.
